requirements.txt

```
@@ -25,3 +25,4 @@ sphinx-mdinclude
 sphinx-prompt
 sphinx_rtd_theme>=1.1.0
 sphinxcontrib-bibtex
+sphinx_book_theme
```

For conda-forge compatibility
```suggestion
sphinx-book-theme
```